corehq/apps/locations/views.py

```diff
@@ -275,7 +275,6 @@ class LocationFieldsView(CustomDataModelMixin, BaseLocationView):
     entity_string = ugettext_lazy("Location")
     template_name = "custom_data_fields/custom_data_fields.html"
 
-    @method_decorator(require_can_edit_locations)
     @method_decorator(locations_access_required)
     @method_decorator(domain_admin_required)
```

Comment:
Doesn't domain admin also imply `locations_access_required`?

Reply:
`locations_access_required` is a domain level privileges check (eg is this a standard plan or better)

Reply:
Just to confirm, isn't it actually user role check, https://github.com/dimagi/commcare-hq/blob/4f0583ac47ae960e863a8edaf40d3139a49e6701/corehq/apps/locations/permissions.py#L146?